pdlua.c

```diff
@@ -1057,22 +1070,31 @@ static t_int *pdlua_perform(t_int *w){
     for (int i = 0; i < o->siginlets; i++)
     {
         lua_newtable(__L());
-        t_float *in = (t_float*)w[i + 3];
-        for (int j = 0; j < nblock; j++)
+        t_signal *sig = (t_signal *)(w[2 + i]);
+        t_float *in = sig->s_vec;
+#if PD_HAVE_MULTICHANNEL
+        int nchans = sig->s_nchans;
```

Comment:
after looking at this once more, i thought this should also have a runtime check - so i'm currently a bit irritated why i get input samples in Pd0.53 with a version compiled with Pd0.55 ...

Reply:
not so irritated anymore. there was no sample table in that case. fixed with ce8a92dcf6a4458f1014a6344b85580ca78be451